packages/core/src/amazonqFeatureDev/errors.ts

```diff
@@ -159,7 +142,10 @@ export class ApiError extends ToolkitError {
     }
 }
 
-export const denyListedErrors: string[] = ['Deserialization error', 'Inaccessible host']
+export const denyListedErrors: string[] = [
+    i18n('AWS.amazonq.featureDev.error.deserialization'),
```

Comment:
Are these strings i18n'd on the backend?

Reply:
Good question. Theoretically should have, but I would not rely on those unless we have some uniformity (or make sure they match both toolkit and service). I'll update to keep those static.

Reply:
Good question. Theoretically should have, but I would not rely on those unless we have some uniformity (or make sure they match both toolkit and service). I'll update to keep those static.